index.js

```diff
@@ -17,6 +17,7 @@ module.exports = function (moduleName, options) {
     rcStrictJson: false,
     stopDir: oshomedir(),
     cache: true,
+    ensureUserHome: true,
```

Comment:
I added it because of the unit tests. I wanted to have a way to control if the original behavior was working the same way.